searx/__init__.py

```diff
@@ -16,9 +16,9 @@
 '''
 
 import logging
+import io
```

Comment:
Why is it better according to pylint? I thought it was always better to import only the functions you are using, not the whole moduel.

Reply:
the previous code was ```from io import open``` which override `open`, pylint is not happy about that.

Actually I think we can use `open` directly.

Reply:
Updated.

Reply:
> Actually I think we can use `open` directly, and just remove `from io import open`.

AFAIK `io.open` was sometimes needed in py2.  In py3 `open` and `io.open` are the same thing.